test/dlc_tests/conftest.py

```diff
@@ -783,7 +783,7 @@ def generate_unique_values_for_fixtures(metafunc_obj, images_to_parametrize, val
                             image,
                             f"{metafunc_obj.function.__name__}-{framework_name_map.get(framework)}-"
                             f"{job_type_map.get(job_type)}-{image_tag}-"
-                            f"{os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')}-{index}{instance_tag}",
+                            f"{os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')}-{str(uuid.uuid4())}{instance_tag}",
```

Comment:
Could we add a "-" in between the uuid value and the instance tag to keep the two separated? Makes the output cleaner and easier to look up if ever needed
`-{str(uuid.uuid4())}{instance_tag}"` -> `-{str(uuid.uuid4())}-{instance_tag}"`

Reply:
+1

Reply:
@ohadkatz `instance_tag` is either `""` or `f"-{instance_type.replace(".", "-")}"`, so the `-` is already a part of the `instance_tag` variable.

Reply:
the `instance_tag` isn't always used, in my case i had `test_dependency_check_cpu-tf-inf-2-9-0-cpu-py39-ubuntu20-04-ec2-pr-2250-2022-09-22-17-12-21-4d2b410b50ccc8a66e1a643a31453cbad1987bf4-1` as key name, since `instance_tag` isn't always available, if we add `-`, this would mean that above key would now become `test_dependency_check_cpu-tf-inf-2-9-0-cpu-py39-ubuntu20-04-ec2-pr-2250-2022-09-22-17-12-21-4d2b410b50ccc8a66e1a643a31453cbad1987bf4-1-`

Reply:
I see, lets keep it as is! Thanks for the info @saimidu